### ISSUE TITLE
Anyways to get this working with ollama

### ISSUE THREAD

**Issue by TheMindExpansionNetwork** [reality-comes/discordrp#1]
Kind of a noonish I really dig what your doing but open ai going to be adding quick when releasing to madness I want to use open source is there ways to do this only with the ollama

**Comment by reality-comes** [comment]
Yes, I just pushed update to dev branch that supports ollama. I added some support for it a while back and never updated. Let me add some documentation for it.

**Comment by reality-comes** [comment]
I added ollama support for embeddings, between that and koboldcpp you can do everything locally. I added documentation on how to switch to it in the readme. To edit ollama settings you will need to change them in embeddings.py (for instance switching the ollama model or the url.)
